tests/data/config/sim_mode_2.yaml

```diff
@@ -1,5 +1,5 @@
 t2sa_ip: "127.0.0.1"
-t2sa_port: 50000
+t2sa_port: 0
```

Comment:
port 0? is this correct?

Reply:
It's correct for tests running in simulation mode 2, where we run our own mock server and don't need to connect to anything non-local. I want that to run on any available port because it runs the tests quickly enough their ports collide otherwise. In simulation mode 0 and 1, we use a static port (50000 is the default)